portfolio/src/main/webapp/script.js

```diff
@@ -81,3 +83,10 @@ function createListElement(comment) {
   liElement.appendChild(pElement);
   return liElement;
 }
+
+// Fetches to delete all comments
+function deleteAllComments() {
+  console.log('/delete-data');
+  fetch('/delete-data')
```

Comment:
Might be cool if this returned some sort of status message, like maybe saying how many records were deleted:  "Successfully deleted 10 comments"